documentation/user_journeys.md

```diff
@@ -0,0 +1,19 @@
+# PacketManager - PrivateChed
+
+## 1. Andrés.
+
+32 años, Desarrollador de software.
+
+Cada lunes me preparo para hacer meal prep para toda la semana. Lo primero que hago es revisar lo que tengo en casa, sobre todo los ingredientes frescos como el pollo y las verduras, y también los productos no perecederos como la pasta y el arroz. Mi objetivo es encontrar recetas que utilicen lo que ya tengo, pero me frustro porque muchas veces las recetas que encuentro requieren ingredientes adicionales o no aprovechan los perecederos que tengo que consumir pronto. A mitad de semana, me doy cuenta de que algunos ingredientes frescos se están echando a perder, y me siento mal por no haber sido más eficiente en la planificación.
```

Comment:
@mmnuria @juanmaaf si os asigno este repo, ¿entendéis todo lo que dice aquí? ¿Podríais empezar a hacer issues o hay alguna cosa que no entendéis?

Reply:
En teoría entiendo que, quiere que la aplicación le ofrezca la posibilidad de que añadiendo los ingredientes que tiene en casa para gastar (de esa manera aprovechar los alimentos que se puedan poner malos ), le sugiera recetas de comida a realizar y asi poder organizarse mejor para que nada se eche a perder.

Reply:
¿Estás teniendo en cuenta lo que menciona sobre los ingredientes frescos?

Reply:
A mitad de semana, además...

Reply:
Si no me equivoco creo que, cuando menciona lo de a mitad de semana, se refiere a lo que quiere evitar es decir, para que no le pase que a mitad de semana se encuentre alimentos echados a perder, quiere una solución para que desde el inicio de la semana las comidas estén planificadas y de esa manera no le ocurra lo que expone que le pasa a mitad de semana.

Es cierto que si que sugeriría que lo redactara un poquito más descriptivo haciendo distinción entre, por un lado pa exposición del problema y por otro lado lo que quiere obtener una vez solucionado su problema, de esa forma, quedaría mejor explicado cada parte.

Reply:
> Si no me equivoco creo que, cuando menciona lo de a mitad de semana, se refiere a lo que quiere evitar es decir, para que no le pase que a mitad de semana se encuentre alimentos echados a perder, quiere una solución para que desde el inicio de la semana las comidas estén planificadas y de esa manera no le ocurra lo que expone que le pasa a mitad de semana.
> 

Si estás diciendo "si no me equivoco" es que la descripción da lugar a equívoco, y por tanto hay que completarla, ¿no? Una metodología en ingeniería tiene que tratar de reducir los equívocos a un mínimo. Si tienes que preguntar, estás haciendo perder el tiempo a la persona que ha creado la HU. Si no preguntas e interpretas de una forma determinada, vas a crear código que no va a servir al cliente y por tanto también estás haciendo uso poco eficiente de los recursos de la empresa.

> Es cierto que si que sugeriría que lo redactara un poquito más descriptivo haciendo distinción entre, por un lado pa exposición del problema y por otro lado lo que quiere obtener una vez solucionado su problema, de esa forma, quedaría mejor explicado cada parte.

:+1: